persistence/registry/appinitdlls/persist-via-appinit_dlls-registry-key.yml

```diff
@@ -9,11 +9,18 @@ rule:
     examples:
       - Practical Malware Analysis Lab 11-02.dll_:0x1000158b
   features:
-    - and:
-      - or:
+    - or:
+      - and:
         - or:
           - match: set registry value
+          - or:
+            - number: 0x80000002 = HKEY_LOCAL_MACHINE
         - or:
-          - number: 0x80000002 = HKEY_LOCAL_MACHINE
-      - string: /SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Windows/i
-      - string: AppInit_DLLs
+          - string: /Software\\Microsoft\\Windows NT\\CurrentVersion\\Windows/i
+          - string: /Software\\Wow6432Node\\Microsoft\\Windows NT\\CurrentVersion\\Windows/i
+        - string: /AppInit_DLLs/i
+      - and:
+        - match: create process
+        - or:
+          - string: /REG ADD[\s\"\\a-z_]{5,20}Software\\Microsoft\\Windows NT\\CurrentVersion\\Windows/i
+          - string: /REG ADD[\s\"\\a-z_]{5,20}Software\\Wow6432Node\\Microsoft\\Windows NT\\CurrentVersion\\Windows/i
```

Comment:
Also, I wonder if we should include the `Software...` strings once instead of duplicating them. Adding a new block with `- string: /REG ADD/i` and the `create process` should work for this.

Reply:
Yeah, this makes sense. The commits 7d550ec and 13e454a should implement your suggestion for all of the rules that are part of my PR.